dangerfile.ts

```diff
@@ -1,259 +1,19 @@
-// inspire by reacts dangerfile
+// Inspired by React dangerfile
```

Comment:
Not anymore, I guess
```suggestion
```

Reply:
It is, but it has nothing in common with ours.

Reply:
Ok, then, let's remove this everywhere.